freq/static/freq/js/buttons.js

```diff
@@ -0,0 +1,149 @@
+/*global $:false, jQuery:false, moment:false, ko:false, fillChart:false, */
+
+
+function datePickerValue() {
+    return {
+        start: $('#date_picker_start').val(),
+        end: $('#date_picker_end').val()
+    };
+}
+
+
+function spinnerValue(i) {
+    return function(){
```

Comment:
Dit is een manier om een soort van Classes te maken. Ik gebruik ze omdat callback functies alleen de response verwerken en geen extra parameters meegegeven kunnen worden. Op deze wijze kan je een aantal parameters instantieren.